tests/manager/manager-ctor-write_concern-005.phpt

```diff
@@ -0,0 +1,41 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): write concern options (64-bit wtimeoutms)
+--FILE--
+<?php
+
+$tests = [
+    ['mongodb://127.0.0.1/?w=2&wtimeoutms=4294967296', []],
+    ['mongodb://127.0.0.1/?w=majority&wtimeoutms=4294967296', []],
+    ['mongodb://127.0.0.1/?w=customTagSet&wtimeoutms=4294967296', []],
+];
+
+foreach ($tests as $test) {
+    list($uri, $options) = $test;
+
+    $manager = new MongoDB\Driver\Manager($uri, $options);
```

Comment:
`$options` is never used, so I think you can just get rid of it and make `$tests` an array of URI strings.